dev-python/pytest-twisted/pytest-twisted-1.13.4.ebuild

```
@@ -0,0 +1,47 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="pytest-twisted is a plugin for pytest that allows you to test code which uses the twisted framework"
+HOMEPAGE="https://github.com/pytest-dev/pytest-twisted"
+SRC_URI="https://github.com/pytest-dev/pytest-twisted/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-python/decorator[${PYTHON_USEDEP}]
+	dev-python/greenlet[${PYTHON_USEDEP}]
+	>=dev-python/pytest-2.3[${PYTHON_USEDEP}]
+	dev-python/twisted[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	# https://github.com/pytest-dev/pytest/issues/9280
+	sed -e '/^pytest_plugins =/d' -i testing/conftest.py || die
+
+	default
```

```suggestion
	distutils-r1_src_prepare
```
As in `src_install`, `distutils-r1_src_install` [would be used in case of absence of `src_prepare`-definition](https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/distutils-r1.eclass#n195).